Global/AlteraQuartusII.gitignore

```diff
@@ -0,0 +1,60 @@
+##list taken from http://www.alterawiki.com/wiki/Version_Control (01.10.2015)
+
+######### Quartus II source files
+
+# project files:
+### project_name.qpf Quartus II project file
```

Comment:
Are we able to bring these comments closer to the source? I think this is a neat list to curate, but currently it's very verbose and the comments being separated from the rules like this makes it hard to follow.